AirCasting/Map/GoogleMap/GoogleMapView.swift

```diff
@@ -0,0 +1,184 @@
+//  Created by Lunar on 26/01/2021.
+//
+import UIKit
+import SwiftUI
+import GoogleMaps
+import Resolver
+
+struct GoogleMapView: UIViewRepresentable {
```

Comment:
Is this map only used to display session location and path? Because if yes, then maybe we could rename it to indicate that? Because I see that it is tightly associated with notes and heatmap logic, so it's not really a generic GoogleMap

Reply:
Yeah, you are right. The generic approach here is used (somehow) in GoogleMapCore which I think can be the base for every other map we will be using/creating (as this one). This map as you said is used for many things - saying that I don't have idead for a better name to be honest. Would like like to still rename it? If so, do you have maybe some ideas?